src/app/view/editor-main-view/data-views/trainrunsections.view.ts

```diff
@@ -1055,6 +1055,7 @@ export class TrainrunSectionsView {
   ) {
     (["BEGINNING_ARROW", "ENDING_ARROW"] as const).forEach((arrowType) => {
       groupLinesEnter
+        .filter((d: TrainrunSectionViewObject) => !d.trainrunSection.getTrainrun().isRoundTrip())
         .append(StaticDomTags.EDGE_LINE_ARROW_SVG)
         .attr("d", (d: TrainrunSectionViewObject) => {
           return d.trainrunSection.getTrainrun().isRoundTrip() ? "" : "M-4,-5L2,0L-4,5Z";
```

Comment:
Then we can even do this

```suggestion
        .filter((d: TrainrunSectionViewObject) => !d.trainrunSection.getTrainrun().isRoundTrip())
        .append(StaticDomTags.EDGE_LINE_ARROW_SVG)
        .attr("d", "M-4,-5L2,0L-4,5Z")
```